guides/common/modules/ref_redhat-cloud-settings.adoc

```diff
@@ -8,8 +8,5 @@
 | *Synchronize recommendations Automatically* | No | Enable automatic synchronization of Insights recommendations from the {Team} cloud.
 | *Obfuscate host names* | No | Obfuscate hostnames sent to the {Team} cloud.
 | *Obfuscate host ipv4 addresses* | No | Obfuscate IPv4 addresses sent to the {Team} cloud.
-| *{Team} Cloud token* | \\***** | Authentication token sent to the {Team} cloud services.
-Used to authenticate requests to cloud APIs.
-| *Exclude installed Packages* | No | Exclude installed packages from upload to the {Team} cloud.
-| *Include parameters in insights-client reports* | No | Include parameter tags from {Project} during import.
+| *ID of the RHC(Yggdrasil) daemon* | \\***** | RHC daemon id.
```

Comment:
This is a case where you have branding: RHC is the Red Hat branded version and Yggdrasil is the unbranded version. In upstream RHC doesn't show up anywhere, but I'm unsure about downstream.

A good example where an attribute is needed.

Reply:
Added the attribute best to my knowledge. If it's good, I can replace it wherever applicable.

Reply:
Hmm, now I look at the source and it's actually also branded in https://github.com/theforeman/foreman_rh_cloud/blob/abe5d7538d22597edb05cc839841fa793f8b5f0d/lib/foreman_rh_cloud/settings.rb#L9 so I may have been wrong that an attribute was needed.

Reply:
I am still in favor of your former decision :sweat_smile: 

However, I am open to your ultimate opinion.

Reply:
Addressed :+1: